physlr/physlr.py

```diff
@@ -1332,6 +1332,79 @@ def determine_molecules_bc_cosine_of_squared(g, u):
                 for community in
                 Physlr.detect_communities_cosine_of_squared(g, bi_connected_component)]
 
+    @staticmethod
+    def partition_subgraph_into_bins_randomly(node_set, max_size=40):
+        """Prtition the subgraph into bins randomly for faster processing. Return bins."""
+        bins_count = 1
+        if len(node_set) > max_size:
+            bins_count = 1 + int(len(node_set) / max_size)
```

Comment:
Replace 1338 to 1340 with
```suggestion
        bins_count = 1 + len(node_set) // max_size
```
Avoid conditionals where they're not necessary.